ui/DesignSystem/Component/Wallet/Transactions/TxListItem.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="typescript">
+  import type { Tx } from "ui/src/transaction";
+  import { Icon } from "ui/DesignSystem/Primitive";
+  import { Label } from "ui/DesignSystem/Component";
+  import { getDay, getShortMonth } from "ui/src/transaction";
+
+  export let tx: Tx;
+
+  const price = 0.0043;
```

Comment:
It will take a bit more effort to get the gas price for transactions. I suggest we remove this for now and add it once we have it. When we merge this we can create a follow-up issue.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1900/commits/908bc169723b3f22c21ad24edb4e589f7efe6fb2